app/views/matches/show.slim

```diff
@@ -38,6 +38,7 @@ link href='http://fonts.googleapis.com/css?family=Permanent+Marker' rel='stylesh
 
     .versus-classes
       .versus-class-pic
+        h3 #{(@players - [@match.oppname]).compact[0]} 
```

Comment:
We can also use ```@match.user.profile.name``` here. This is because sometimes the match_json will not be available and thus ```@players``` will not be populated.